test/mocks/redis/redis.go

```diff
@@ -1,227 +1,151 @@
-// Automatically generated by MockGen. DO NOT EDIT!
-// Source: github.com/envoyproxy/ratelimit/src/redis (interfaces: RateLimitCache,Pool,Connection,Response,TimeSource,JitterRandSource)
+// Code generated by MockGen. DO NOT EDIT.
```

Comment:
Looks like this text changed in mockgen 3 years ago (https://github.com/golang/mock/commit/c9cc560280e96593c64087bea69b082fd7460156) prior to the v1.0.0 mockgen release. I may send out a follow-up to try to make sure everyone is using the same mockgen version even if they're not using `make bootstrap`.